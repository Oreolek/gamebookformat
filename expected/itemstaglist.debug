BEGIN DEBUG OUTPUT
Book title: Items
Number of sections: 3
Info
 This file is for testing the list option. This section should not be listed. Using the options given in the options file for this gamebook, only the sections tagged as todo will be listed. 

Turn to 1 to begin.
1 (end_todo) -  There has to be a section set as missingto (ie this section), otherwise generating a list with only todo sections will fail (as one of them references non-todo sections). Unfortunate side-effect is that this section will always be included in the list. TODO figure out some fix for this.
2 (curse) -  Need to write this section. Sorry for it being a dead-end for now. 
3 (tjunction) -  (This section needs some work!) Here you find a [FOUND item]key[/FOUND] and a [FOUND item]stick[/FOUND]. You can go west to 1, or east to 2. 
END DEBUG OUTPUT
