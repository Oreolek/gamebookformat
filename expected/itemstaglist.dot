digraph gamebook {
  3->1
3->2
}
