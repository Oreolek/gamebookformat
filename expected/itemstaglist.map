{"tjunction": 3, "end_todo": 1, "curse": 2, "start": 1}