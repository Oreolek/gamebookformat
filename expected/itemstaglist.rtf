{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Items
\b0\
\
\b \qc Info
\b0\



\ql  This file is for testing the list option. This section should not be listed. Using the options given in the options file for this gamebook, only the sections tagged as todo will be listed.  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  There has to be a section set as missingto (ie this section), otherwise generating a list with only todo sections will fail (as one of them references non-todo sections). Unfortunate side-effect is that this section will always be included in the list. TODO figure out some fix for this. \
\

\b \qc 2
\b0\
\ql  Need to write this section. Sorry for it being a dead-end for now.  \
\

\b \qc 3
\b0\
\ql  (This section needs some work!) Here you find a key and a stick. You can go west to \b 1
\b0
, or east to \b 2
\b0
.  \
\

}
