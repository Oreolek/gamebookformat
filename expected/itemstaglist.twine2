<tw-storydata name="Items" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> There has to be a section set as missingto (ie this section), otherwise generating a list with only todo sections will fail (as one of them references non-todo sections). Unfortunate side-effect is that this section will always be included in the list. TODO figure out some fix for this.</tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> Need to write this section. Sorry for it being a dead-end for now. </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> (This section needs some work!) Here you find a *key* and a *stick*. You can go west to [[1]]
, or east to [[2]]
. </tw-passagedata>
</tw-storydata>
