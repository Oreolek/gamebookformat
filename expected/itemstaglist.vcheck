examples/itemstaglist.debug: No start section found. Cancelling checks.
