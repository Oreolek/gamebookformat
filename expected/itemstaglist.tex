\documentclass[a5paper,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{Items}
\author{Pelle Nilsson}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\clearpage

\thispagestyle{empty}

\pagestyle{empty}
\subsection*{\begin{center} \textbf{Info} \end{center}}


 \noindent
 This file is for testing the list option. This section should not be listed. Using the options given in the options file for this gamebook, only the sections tagged as todo will be listed. 
\vspace{1em}

Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 There has to be a section set as missingto (ie this section), otherwise generating a list with only todo sections will fail (as one of them references non-todo sections). Unfortunate side-effect is that this section will always be included in the list. TODO figure out some fix for this.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 Need to write this section. Sorry for it being a dead-end for now. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 (This section needs some work!) Here you find a \textbf{key} and a \textbf{stick}. You can go west to \textbf{\autoref{section1}}, or east to \textbf{\autoref{section2}}. 
\vspace{1em}
\end{document}
